amd/src/tribute.js

```diff
@@ -598,19 +598,15 @@ if (!Array.prototype.find) {
             } else {
               coordinates = this.getContentEditableCaretPosition(info.mentionPosition);
             }
-
-            if (document.getElementById('id_messageeditable')) {
-              var topposition = $('#id_messageeditable').offset().top+24
-            } else if (document.getElementsByClassName('.hsuforum-textarea')) {
-              var topposition = $('.hsuforum-textarea').offset().top+24
-            }
+            // Add position of screen to top of the page to coordinates top
+            coordinates.top = document.body.scrollTop + coordinates.top;
 
             // Move the button into place.
-            this.tribute.menu.style.cssText = 'top: ' + topposition + 'px;\n                                           left: ' + coordinates.left + 'px;\n                                           position: absolute;\n                                           zIndex: 10000;\n                                           display: block;';
+            this.tribute.menu.style.cssText = 'top: ' + coordinates.top + 'px;\n left: ' + coordinates.left + 'px;\n position: absolute;\n zIndex: 10000;\n display: block;';
 
-            setTimeout(function () {
-              _this7.scrollIntoView(_this7.getDocument().activeElement);
-            }, 0);
+            // setTimeout(function () {
```

Comment:
I made a comment about this in the description. (Are you not reading the description ;) )

"I also commented out the scrollIntoView functionality as it didn't apply to how it was now working. I didn't delete it as it's part of the tribute code so thought it would be better to keep it there for future reference."

That's why I kept it.